### DISCUSSION TITLE
Add an option to hide/replace the windows audio icon.

### DISCUSSION THREAD

**Discussion by Chaython** [File-New-Project/EarTrumpet#1009] [Ideas]
Add an option to hide/replace the windows audio speaker icon.
With this installed there's simply two speaker icons.
Please add a way to disable windows built in one.

**Comment by almanaccare** [comment]
Hey @Chaython - This is unnecessary if using Windows 10. You can hide the default Windows audio taskbar icons and show the EarTrumpet icon instead by toggling them in Windows taskbar settings. Right click your taskbar, click "Taskbar Settings". Under "Notification area", choose "Select which icons appear on the taskbar".

I'm unsure if the same can be said for Windows 11 but you haven't specified which OS you're using.

**Reply by Chaython** [reply]
![image](https://user-images.githubusercontent.com/6486343/149186396-678fb55b-c71c-4250-afb2-df32b5d1cd41.png)
Ya, not an option in win11

**Reply by almanaccare** [reply]
That's unfortunate. Hopefully a fix incoming soon then.

**Reply by Chaython** [reply]
This group policy seems ignored.
There's still a group policy HideSCAVolume
Which amends Computer\HKEY_USERS\S-1-5-21-2388616930-3115959047-328434320-1017\Software\Microsoft\Windows\CurrentVersion\Group Policy Objects\{A71D1DC4-3F00-4EDC-A238-8465C1EB6289}User\Software\Microsoft\Windows\CurrentVersion\Policies\Explorer
Though user number is obviously wild

**Reply by Chaython** [reply]
The group policy is ignored. The   Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Policies\Explorer HideSCAVolume[dword] is also ignored. At least in my system...
https://admx.help/?Category=Windows_10_2016&Policy=Microsoft.Policies.TaskBar2::HideSCAVolume

**Reply by riverar** [reply]
Yep, there's no way to hide the default icon in Windows 11 at this time. Please file feedback in Feedback Hub! By flooding Microsoft with requests, hopefully we can convince them to make a change here.

**Reply by Chaython** [reply]
The volume icon is now the action center button, combining other applets such as MS Edge's video player. 
![image](https://user-images.githubusercontent.com/6486343/149458635-2065fee4-010c-4758-a8e4-43dc1b467eef.png)
So likely the best choice is to make a revision with its own action center widget?

**Reply by riverar** [reply]
@Chaython This space is not customizable by third parties.

**Comment by CaptainFrosty** [comment]
This is limited to Windows 11 I believe, and me unfortunately. The system icons have been separated from the taskbar drawer and are now included in an adjacent module called Windows.UI.QuickActions.dll.
